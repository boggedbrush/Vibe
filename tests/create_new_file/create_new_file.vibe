# VibeSpec: 1.6
patch_type: add_block
file: new_file.py
position: start
--- code: |
  import numpy as np
  import matplotlib.pyplot as plt

  print("new_file.py created and imports added.")