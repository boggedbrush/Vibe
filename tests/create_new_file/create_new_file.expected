import numpy as np
import matplotlib.pyplot as plt

print("new_file.py created and imports added.")
